vendor/engines/c2po/app/views/c2po/facility_accounts/show/_remittance_information.html.haml

```diff
@@ -0,0 +1,2 @@
+- if @account.is_a?(CreditCardAccount) || @account.is_a?(PurchaseOrderAccount)
+  = f.input :remittance_information, value_method: Proc.new { |value| value.present? ? simple_format(value) : "None Entered"}
```

Comment:
A nitpicky spacing issue at the end of this line (I see it was like that all along)